packages/@uppy/companion/src/server/provider/index.js

```diff
@@ -9,11 +9,29 @@ const instagram = require('./instagram')
 const instagramGraph = require('./instagram/graph')
 const facebook = require('./facebook')
 const onedrive = require('./onedrive')
+const zoom = require('./zoom')
 const { getURLBuilder } = require('../helpers/utils')
 const logger = require('../logger')
 // eslint-disable-next-line
 const Provider = require('./Provider')
 
+// leave here for now until Purest Providers gets updated with Zoom provider
```

Comment:
I opened a pr here https://github.com/simov/purest-providers/pull/1 but it doesn't look like Purest providers has been updated in a few years. this was required to get the auth flow working